pkg/gitrepo/repo_test.go

```diff
@@ -251,6 +279,21 @@ func Test_Repo_ResolveVersion(t *testing.T) {
 			var err error
 			var version string
 
+			// Set HEAD.
+			{
+				ref := plumbing.NewReferenceFromStrings(plumbing.HEAD.String(), tc.inputHeadTarget)
+				err := repo.storage.SetReference(ref)
+				if err != nil {
+					t.Fatalf("err = %v, want %v", microerror.Stack(err), nil)
+				}
+			}
+
+			h, err := repo.HeadSHA(context.Background())
+			if err != nil {
+				t.Fatalf("err = %v, want %v", microerror.Stack(err), nil)
+			}
+			t.Logf("head=%s", h)
```

Comment:
What is with that?

Reply:
My debugging leftover. I'm removing it. Thanks.